imap_processing/spice/geometry.py

```diff
@@ -62,6 +62,24 @@ class SpiceFrame(IntEnum):
     IMAP_GLOWS = -43750
 
 
+# TODO: Update boresight for in-situ instruments
+# TODO: Confirm ENA boresight vectors
+BORESIGHT_LOOKUP = {
```

Comment:
This is my low effort initial approach to defining instrument boresights. I am open to suggestions of how to make this better, where to move this definition, etc. I still don't know if we should eventually create an instrument kernel that defines these. I need to get a better understanding of all of the requirements from all of the instruments before making that decision.

Reply:
Do we have to use `.name` or can it just be the enumeration itself as the lookup?